frappe/cache_manager.py

```diff
@@ -22,6 +23,10 @@
 doctype_cache_keys = ("meta", "form_meta", "table_columns", "last_modified",
 		"linked_doctypes", 'notifications', 'workflow' ,'energy_point_rule_map')
 
+count_cache_blacklist = ["Version", "Tag", "ToDo", "List Filter", "Note Seen By", "Notification Log",
+		"Document Follow", "Communication", "Email Queue", "Deleted Document", "File", "Email Queue Recipient"
+		"Comment", "Has Role", "Attendance", "Route History"]
```

Comment:
why are ToDo, Communication & File in this list?

Reply:
These doctypes are frequently entered having cache rebuild on these will be an unnecessary overhead. The cache is not reliable in terms of accuracy it is just used to check of doctype contains a record or not, for desk only.